firebase_admin/_auth_utils.py

```diff
@@ -181,3 +182,74 @@ def validate_custom_claims(custom_claims, required=False):
         raise ValueError(
             'Claim "{0}" is reserved, and must not be set.'.format(invalid_claims.pop()))
     return claims_str
+
+def validate_action_code_settings(settings):
```

Comment:
Lets call this `encode_action_code_settings()` to be consistent with the terminology used in FCM. I would also consider moving this to the same file where ActionCodeSettings is defined.